src/views/ModelPlan/ReadOnly/GeneralCharacteristics/index.tsx

```diff
@@ -285,99 +366,140 @@ const ReadOnlyGeneralCharacteristics = ({
+          />
+        )}
 
-          {rulemakingRequired && (
-            <div className="desktop:width-card-lg">
-              <ReadOnlySection
-                heading={t('ruleMakingInfo')}
-                copy={rulemakingRequiredDescription}
-              />
-            </div>
-          )}
-        </div>
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'geographyType',
+          <ReadOnlySection
+            heading={t('geographyType')}
+            list
+            listItems={geographiesTargetedTypes?.map(translateGeographyTypes)}
+            listOtherItem={geographiesTargetedTypesOther}
+          />
+        )}
+
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'geographyApplied',
+          <ReadOnlySection
+            heading={t('geographyApplied')}
+            list
+            listItems={geographiesTargetedAppliedTo?.map(
+              translateGeographyApplication
+            )}
+            listOtherItem={geographiesTargetedAppliedToOther}
+            notes={geographiesTargetedNote}
+          />
+        )}
+
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'participationOptions',
+          <ReadOnlySection
+            heading={t('participationOptions')}
+            copy={translateBooleanOrNull(participationOptions)}
+            notes={participationOptionsNote}
+          />
+        )}
+
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'agreementType',
+          <ReadOnlySection
+            heading={t('agreementType')}
+            list
+            listItems={agreementTypes?.map(translateAgreementTypes)}
+            listOtherItem={agreementTypesOther}
+          />
+        )}
+
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'moreParticipation',
+          <ReadOnlySection
+            heading={t('moreParticipation')}
+            copy={translateBooleanOrNull(multiplePatricipationAgreementsNeeded)}
+            notes={multiplePatricipationAgreementsNeededNote}
+          />
+        )}
+      </div>
+      <div>
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'rulemakingRequired',
+          <SideBySideReadOnlySection
+            firstSection={{
+              heading: t('rulemakingRequired'),
+              copy: translateBooleanOrNull(rulemakingRequired)
+            }}
+            secondSection={
+              rulemakingRequired === true && {
+                heading: t('ruleMakingInfo'),
+                copy: rulemakingRequiredDescription
+              }
+            }
+          />
+        )}
         {rulemakingRequiredNote && (
           <ReadOnlySection
             heading={t('basics:notes')}
             copy={rulemakingRequiredNote}
           />
         )}
```

Comment:
Another instance where this component will always render in every view if a note exists